target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -109,6 +96,14 @@
 					bias-disable;
 				};
 			};
+
+			usbvcc_pins: usbvcc_pins {
+				pins = "gpio23", "gpio25";
+				function = "gpio";
+				drive-strength = <2>;
```

Comment:
In c2600 gpl pins 23 and 25 refer to
static struct gpiomux_setting gsbi2_spi_data_cfg = {
	.func = GPIOMUX_FUNC_GPIO,
	.drv = GPIOMUX_DRV_10MA,
	.pull = GPIOMUX_PULL_UP,
	.dir = GPIOMUX_OUT_HIGH,
};

Weird, maybe tp-link haven't bothered to make another def, but it seems that drive-strength should be 10